source/Nuke.Common/ProjectModel/SolutionAttribute.cs

```diff
@@ -53,7 +53,9 @@ public override object GetValue(MemberInfo member, object instance)
                                GetSolutionFileFromParametersFile(member);
             var deserializer = typeof(SolutionSerializer).GetMethod(nameof(SolutionSerializer.DeserializeFromFile)).NotNull()
                 .MakeGenericMethod(member.GetMemberType());
-            return deserializer.Invoke(obj: null, new object[] { solutionFile });
+            if (File.Exists(solutionFile))
```

Comment:
Pre-existing bug - this was crashing if it couldn't find the solution file

Reply:
you can repro by:
1) adding a new target to [Build.cs](https://github.com/nuke-build/nuke/blob/develop/build/Build.cs):
```
    Target TestIt => _ => _
        .Executes(() =>
        {
            Console.WriteLine("Hello");
        });
```

2) running `dotnet run --project build/_build.csproj TestIt`

Gives you this:
![image](https://user-images.githubusercontent.com/373389/163055215-3ce39a7f-66c0-42c6-9f73-fe7e10e24bbd.png)

With this change, it no longer logs that exception:
![image](https://user-images.githubusercontent.com/373389/163055274-ed03c96e-818c-4c6b-a237-80345d2069a9.png)

The question is though, should it throw a exception though...

Seems to come from [Build.GlobalSolution.cs](https://github.com/nuke-build/nuke/blob/develop/build/Build.GlobalSolution.cs#L25), which does some magic to create a file, it seems.

So we can either:
* throw a more obvious exception
* Change the code in [Build.GlobalSolution.cs](https://github.com/nuke-build/nuke/blob/develop/build/Build.GlobalSolution.cs#L25) to not expect the solution file to already exist
* return null, as this change does